ytdlp_handler.py

```diff
@@ -151,7 +170,11 @@ def download_progress_bar(d):
         progress_percent = (
             "-" if downloaded == "-" or total == 0 else int(downloaded / total * 100)
         )
-        DL_PROGRESS_WINDOW["PROGINFOS1"].update(f"{progress_percent}%")
+
+        playlist_index = 1 if not d['info_dict']['playlist_index'] else d['info_dict']['playlist_index']
+        n_entries = 1 if not d['info_dict']['playlist_index'] else d['info_dict']['n_entries']
```

Comment:
Set a global variable `VIDEO_INDEX` and set it to 1 then use it here

Reply:
I have remarks for this [here](https://github.com/Kenshin9977/video-dl/pull/10#discussion_r777223315)